content/metastore-integrations/docs/hive-metastore.md

```diff
@@ -157,24 +156,12 @@ aws emr-containers start-job-run --cli-input-json file:///Spark-Python-in-s3-hms
 
 ```
 
```

Comment:
Thank you @melodyyangaws for this fantastic PR. I think there are some disruptive changes from previous commit? If so, can you please do rebase with upstream main and re-submit PR?

Reply:
done. thx Peter.